Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -111,6 +117,30 @@ async function run() {
                 await swapSlot(endPoint, resourceGroupName, webAppName, sourceSlot, swapWithProduction, targetSlot, preserveVnet);
                 break;
             }
+            case "Enable Continuous Monitoring": {
+                var appInsightsManage = new appInsightsManageUtils.AppInsightsManage(endPoint, appInsightsResourceGroupName, appInsightsResourceName, webAppName, resourceGroupName, specifySlotFlag, slotName);
+                await appInsightsManage.configureAppInsights();
+                break;   
+            }
+            case "Start Webjobs": {
+                resourceGroupName = (specifySlotFlag ? resourceGroupName : await azureRmUtil.getResourceGroupName(endPoint, webAppName));
+                var publishingProfile = await azureRmUtil.getAzureRMWebAppPublishProfile(endPoint, webAppName, resourceGroupName, specifySlotFlag, slotName);
+                var continuousJobs = await azureRmUtil.getAllContinuousWebJobs(publishingProfile);
+                for(var continuousJob of continuousJobs) {
+                    await azureRmUtil.startContinuousWebJob(publishingProfile, continuousJob.name);
```

Comment:
Can we add debug statements before and after actions?
This can help handy to debug any issue.
Also print the response in debug mode.

Reply:
@kmkumaran , Just a suggestion, Should we add Stop WebJobs / Start WebJobs as check-box under Start / Stop App Service? Cx may find managing the release complex.
And do we want to update the stop / start webjobs in Kudu Deployments (the call we make to update the status after deployment and manage actions)?

Reply:
@vincentdass - Checkbox is a good suggestion. @Ajay-MS - can you check with @RoopeshNair on this.